source/support.tex

```diff
@@ -624,7 +624,7 @@
 #define @\defnlibxname{cpp_lib_filesystem}@                        201703L // also in \libheader{filesystem}
 #define @\defnlibxname{cpp_lib_flat_map}@                          202207L // also in \libheader{flat_map}
 #define @\defnlibxname{cpp_lib_flat_set}@                          202207L // also in \libheader{flat_set}
-#define @\defnlibxname{cpp_lib_format}@                            202207L // also in \libheader{format}
+#define @\defnlibxname{cpp_lib_format}@                            202306L // also in \libheader{format}
```

Comment:
We have several papers from Varna that all bump this macro, LWG and the SG10 chair proposed that we use distinct values for them, to aid implementers who might not implement them all at once. This paper is the smallest and simplest, so the value 202304 would make sense here (and 202305 for P2757R3, and 202306 for P2637R3).

Reply:
See the notes in https://isocpp.org/files/papers/P2757R3.html#feature-test-macro and https://isocpp.org/files/papers/P2637R3.html#feature-test-macro

Reply:
Fixed.